controller/controller.go

```diff
@@ -139,12 +139,12 @@ func (c *Controller) Boot() {
 		}
 
 		notifier := func(err error, d time.Duration) {
-			c.logger.LogCtx(ctx, "function", "Boot", "level", "warning", "message", "retrying controller boot due to error", "stack", fmt.Sprintf("%#v", err))
+			c.logger.LogCtx(ctx, "level", "warning", "message", "retrying controller boot due to error", "stack", fmt.Sprintf("%#v", err))
```

Comment:
We talked about this recently and the consensus so far was that we want to drop the `function` key in the logs because they do not seem to be super useful to us.